libs/ledger-live-common/src/families/evm/__tests__/rpc.unit.test.ts

```diff
@@ -1,9 +1,9 @@
 import { AssertionError } from "assert";
-import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { CryptoCurrency, CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
 import { DEFAULT_RETRIES_RPC_METHODS, withApi } from "../api/rpc.common";
 
 const fakeCurrency: Partial<CryptoCurrency> = {
-  id: "my_new_chain",
+  id: "my_new_chain" as CryptoCurrencyId,
```

Comment:
Does this work? As `my_new_chain` isn't part of the defined IDs.

Reply:
It does work indeed, it's a way to make typescript compiler consider `my_new_chain` as a CrytoCurrencyId